src/main/drivers/rangefinder_hcsr04.h

```diff
@@ -19,4 +19,6 @@
 
 #include "drivers/rangefinder.h"
 
+#define RANGEFINDER_HCSR04_TASK_PERIOD_MS 50
```

Comment:
There's also an US-100 sonar (very good), which is also very slow - requires at least 100ms measurement cycle to work reliably. Probably it would be better if we make this period configurable.